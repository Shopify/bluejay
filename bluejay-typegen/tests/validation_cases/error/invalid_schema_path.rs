#[bluejay_typegen::typegen("")]
mod schema {}

fn main() {}
