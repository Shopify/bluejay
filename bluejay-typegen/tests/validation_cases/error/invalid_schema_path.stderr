error: Is a directory (os error 21)
 --> tests/validation_cases/error/invalid_schema_path.rs:1:28
  |
1 | #[bluejay_typegen::typegen("")]
  |                            ^^
